textrepo-app/src/test/java/nl/knaw/huc/service/index/MappedIndexerTest.java

```diff
@@ -85,7 +86,11 @@ public void index_usesMultipartFormData_whenFieldsTypeIsMultipart() throws Index
         .withMethod("POST")
         .withPath("/fields")
         .withHeader("Content-Type", "multipart/form-data.*")
-        .withBody(regex("(.|\n|\r)*Content-Type: text/plain(.|\n|\r)*name=\"file\"(.|\n|\r)*latest version contents(.|\n|\r)*"));
+        .withBody(regex("(.|\n|\r)*" +
```

Comment:
Volstaat regex(".*") niet? '.' staat voor any character, dus ook voor \n en \r dan toch?
Ik dacht: .* means "zero or more instances of any character"
Als dit klopt, kunnen deze en vergelijkbare regex'n nog iets vereenvoudigd.

Reply:
Nope, helaas, mock-server matcht in een json-string waarin de line breaks als `\n\r` staan uitgeschreven.